snapcraft.yaml

```diff
@@ -296,12 +296,8 @@ parts:
     plugin: nil
     build-packages:
       - g++
-      - on amd64:
-        - acpica-tools
-        - uuid-dev
-      - on arm64:
-        - acpica-tools
-        - uuid-dev
+      - acpica-tools
```

Comment:
wont this cause these packages to be unnecessarily installed on some archs?

Reply:
Those are in the build-packages section so presumably they will be installed on all arches (same as the `g++` one) but after looking at https://packages.ubuntu.com/focal/acpica-tools, it seems it was not available on `riscv64` so probably why this was made arch-dependent.

Now this is no longer applicable on core22 as all 3 packages are available on all arches. So they will be installed but the part as a whole will no do anything for some arches. Those packages' bits won't end up in the final snap as they are in the build-packages section.

So I think this is OK as it is, LP builders might prove me wrong though.

Reply:
mmm ok cool.

Reply:
With the above said, some time could be saved by (re-)introducing the `- on amd64,arm64` bit in the build-packages so we skip the installation of those 3 packages for say riscv64.

Will do that in a follow-up PR, thanks for bringing that up!